include/simdjson/inline/parsedjson_iterator.h

```diff
@@ -218,7 +218,7 @@ dom::parser::Iterator::Iterator(const dom::parser &pj) noexcept(false)
 #if SIMDJSON_EXCEPTIONS
   if (!pj.valid) { throw simdjson_error(pj.error); }
 #else
-  if (!pj.valid) { abort(); }
+  if (!pj.valid) { return; } //  abort() usage is forbidden in the library
```

Comment:
If this weren't legacy code we plan on removing, I'd be worried this will create code that fails in super unexpected ways. But then, this antipattern of a fallible constructor is part of why this is legacy :)

Reply:
Right. This abort was kind of a hard "assert" anyhow. It would only happen following a programmer's error.